src/map/clif.cpp

```diff
@@ -5242,6 +5244,12 @@ int clif_damage(struct block_list* src, struct block_list* dst, t_tick tick, int
 		}
 	}
 
+	// Calculate what sdelay to send to the client so it applies damage at the same time as the server
+	if (battle_config.synchronize_damage && src->type == BL_MOB && status_get_clientamotion(src) != 0) {
+		// The client only accepts values between 0 and 432; 432 stands for 1x animation speed
```

Comment:
Your comment does not make sense to me.
I tried to understand it with the first monster (SCORPION).
AttackMotion 864 (=sdelay) * DEFAULT_ANIMATION_SPEED (=432) / ClientAttackMotion (=384) => 972
But 972 is out of the range of 0-432.

Reply:
It's fine to send a higher value, it will still just be 1x animation speed.
But maybe I should explain that here.